mathesar/api/exceptions/error_codes.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum, unique
+
+
+@unique
+class ErrorCodes(Enum):
+    NonClassifiedError = 4999
```

Comment:
I think `UnknownError` might be a bit clearer for this.

Reply:
This is more like saying we are able to get the detail of the error but haven't specified any error code for it.

Reply:
Maybe `UnspecifiedCodeError`, then?

Reply:
That was my choice at first but felt like `error code` hasn't been classified yet made sense, `UnspecifiedCodeError` seems like error code has not been specified and there is an error related to that, but I think I will go with `UnknownError` seems neutral.